core/platform/cache/redis_cache_services.py

```diff
@@ -22,10 +22,22 @@
 from core.domain import caching_domain
 import feconf
 import python_utils
+
 import redis
 
-REDIS_CLIENT = redis.Redis(
-    host=feconf.REDISHOST, port=feconf.REDISPORT)
+# Redis client for our own implementation of caching.
+OPPIA_REDIS_CLIENT = redis.StrictRedis(
+    host=feconf.REDISHOST,
+    port=feconf.REDISPORT,
+    db=feconf.OPPIA_REDIS_DB
```

Comment:
Not sure about the constant's name -- should it be feconf.OPPIA_REDIS_DB_NAME instead if it's supposed to be a string (rather than the actual db)? Ditto for the cloud NDB client.

Reply:
It is not a string, but a number, so I have changed it to `OPPIA_REDIS_DB_NUMBER`.